resolwe/flow/executors/__init__.py

```diff
@@ -140,6 +147,16 @@ def run(self, data_id, script, verbosity=1):
 
                         for obj in iterjson(line[3:].strip()):
                             spawn_processors.append(obj)
```

Comment:
This should include the original `name` and random `temp_name`. Then the `hydrate_spawned_files` would not be needed anymore.